airflow/dags/create_external_tables/sentry_error_logs.yml

```diff
@@ -0,0 +1,164 @@
+operator: operators.ExternalTable
+bucket: "{{ env_var('CALITP_BUCKET__SENTRY_EVENTS') }}"
+suffix_bucket: true
+source_objects:
+  - "*.jsonl.gz"
+destination_project_dataset_table: "external_sentry_error_logs.events"
```

Comment:
could you rename `external_sentry_error_logs` to just `external_sentry`? It looks like the dbt source is called `sentry_external_tables`

Reply:
Resolved in [4bb81fe](https://github.com/cal-itp/data-infra/pull/2345/commits/4bb81fed9ac695e1efb83249964b7b990baab2bc)